src/scheduler/workers/unpublish/weekly/sendEmailToNonRespondents.ts

```diff
@@ -11,7 +11,8 @@ export async function sendEmailsToNonRespondents(list: ListWithCountryName) {
     throw new Error(`Exiting for list ${list.id}, not enough information to continue`);
   }
 
-  if (meta.weeksUntilUnpublish >= 6) {
+  // @ts-ignore
+  if (meta?.weeksUntilUnpublish >= 6) {
```

Comment:
meta should not be undefined at this point.

this is why I prefer guard clauses in typescript. typescript is able to detect if you've handled an undefined type (i.e. if(!meta) { throw }).